js/src/logger.ts

```diff
@@ -562,7 +588,7 @@ function logFeedbackImpl(
         [IS_MERGE_FIELD]: true,
       };
     });
-    _state.globalBgLogger().log([record]);
+    state.globalBgLogger().log([record]);
```

Comment:
maybe we should change the method `globalBgLogger` to just `bgLogger`, since it's not necessarily a global itself anymore? And maybe adjust the toplevel comment [here](https://github.com/braintrustdata/braintrust-sdk/blob/91d79548574f3b5a721a10d5e753e1d003da0a66/js/src/logger.ts#L928).

Reply:
done